podman-desktop

```diff
@@ -18,8 +18,52 @@ die() {
   exit 1
 }
 
-command -v multipass >/dev/null 2>&1 || die "missing multipass"
-command -v podman >/dev/null 2>&1 || die "missing podman"
+brew() {
+  set +e
+  if [ -z "$(command -v brew)" ]; then
+    echo "# Install homebrew for easier MacOS management"
+    # shellcheck disable=SC2016
+    echo '/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install.sh)"'
+  fi
+  set -e
+  echo "brew install $1"
+}
+
+validate() {
+  cmd=$1
+  if [ ! -x "$(command -v "$cmd")" ]; then
+    echo "ERROR: \`$cmd\` command not found. Please install $cmd." >&2
+    echo "Instructions are:"
+    if [ "$GOOS" = "linux" ]; then
+      echo "apt-get install $cmd"
+    elif [ "$GOOS" = "darwin" ]; then
+      brew "$cmd"
+    else
+      echo "missing $cmd"
+    fi
+    exit 1
+  else
+    echo "$cmd found."
+  fi
+}
+
+GOOS=$(uname -s | sed s/Darwin/darwin/ | sed s/Linux/linux/)
+
+validate "podman"
+validate "python3"
+if [ ! -x "$(command -v podman-compose)" ]; then
+  echo "ERROR: \`podman-compose\` command not found. Please install podman-compose." >&2
+  echo "Instructions are: python3 -mpip install podman-compose"
+  exit 1
+else
+  echo "podman-compose found."
+fi
+if [ "$GOOS" = "linux" ]; then
+  echo "podman setup complete for Linux"
+  exit 0
+fi
```

Comment:
I'd rather this remain either macOS specific, or as OS-agnostic as possible and not become a general purpose 'setup everything I might want for different distributions' script. If I understood better the goal for a linux port, maybe a second file in the repo would make sense.

Reply:
Considering how little things there are to make this work on Linux, I guess the best solution would be to drop OS-related material from here completely.